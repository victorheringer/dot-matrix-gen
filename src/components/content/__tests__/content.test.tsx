import { render } from "@testing-library/react";
import { Content } from "../index";

describe("Content", () => {
  it("should render correctly", () => {
    const tree = render(
      <Content>
        <p>test</p>
      </Content>
    ).asFragment();
    expect(tree).toMatchSnapshot();
  });
});
