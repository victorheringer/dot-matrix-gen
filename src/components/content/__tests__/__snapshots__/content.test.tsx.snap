// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Content should render correctly 1`] = `
<DocumentFragment>
  <div
    class="sc-bczRLJ lhVdcE"
  >
    <p>
      test
    </p>
  </div>
</DocumentFragment>
`;
